src/Microsoft.ML.Data/Transforms/NormalizerCatalog.cs

```diff
@@ -4,41 +4,40 @@
 
 using Microsoft.ML.Runtime;
 using Microsoft.ML.Runtime.Data;
-using System;
-using System.Collections.Generic;
-using System.Text;
+using Microsoft.ML.Transforms.Normalizers;
 
 namespace Microsoft.ML
 {
     /// <summary>
     /// Extensions for normalizer operations.
     /// </summary>
-    public static class NormalizerCatalogExtensions
+    public static class NormalizerCatalog
     {
         /// <summary>
         /// Normalize (rescale) the column according to the specified <paramref name="mode"/>.
         /// </summary>
         /// <param name="catalog">The transform catalog</param>
-        /// <param name="columnName">The column name</param>
-        /// <param name="mode">The normalization mode (<see cref="Normalizer.NormalizerMode"/>). </param>
-        public static Normalizer Normalizer(this TransformsCatalog catalog, string columnName, Normalizer.NormalizerMode mode = Runtime.Data.Normalizer.NormalizerMode.MinMax)
-            => new Normalizer(CatalogUtils.GetEnvironment(catalog), columnName, mode);
+        /// <param name="inputName">The column name</param>
+        /// <param name="outputName">The column name</param>
+        /// <param name="mode">The <see cref="NormalizingEstimator.NormalizerMode"/> used to map the old values in the new scale. </param>
+        public static NormalizingEstimator Normalizer(this TransformsCatalog catalog, string inputName, string outputName = null, NormalizingEstimator.NormalizerMode mode = NormalizingEstimator.NormalizerMode.MinMax)
```

Comment:
Should this have a verb name instead? `Normalizer` is a noun, and typically methods are verbs - `Normalize`.